cognite_toolkit/_cdf_tk/resource_classes/__init__.py

```diff
@@ -9,6 +9,7 @@
 
 from .asset import AssetYAML
 from .base import BaseModelResource, ToolkitResource
+from .containers import ContainerYAML
```

Comment:
Keep this as is, but I suggest doing this in a separate PR for multiple YAML resources to avoid merge conflicts when you make several of these types of PR in parallel.